### PR TITLE
fixing issue Native libraries list #3896 by adding to integrations table

### PR DESCRIPTION
**Pull request by mgurrie**
added azure and masstransit to integrations this also includes changes from pr added azure and masstransit to integrations for natively instrumented libraries #4308 fixing issue Native libraries list https://github.com/open-telemetry/opentelemetry.io/issues/3896 by adding to integrations table

### CODE REVIEW

**Comment by mgurrie** [comment]
I have signed the CLA let me know if I didn't do it right or need to select something different.

**Comment by svrnm** [comment]
> I have signed the CLA let me know if I didn't do it right or need to select something different.

looks good now:-)

File: data/ecosystem/integrations.yaml [hunk 1]
```diff
@@ -206,3 +206,13 @@
   docsUrl: https://www.krakend.io/docs/telemetry/opentelemetry/
   components: [Go]
   oss: true
+- name: Azure SDKs
+  url: https://github.com/Azure/azure-sdk-for-net
+  docsUrl: https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-enable?tabs=aspnetcore
+  components: [.NET]
+  oss: true
+  - name: MassTransit .NET client
```
Comment (svrnm) [inline data/ecosystem/integrations.yaml:214]:
```suggestion
- name: MassTransit .NET client
```

File: data/ecosystem/integrations.yaml [hunk 2]
```diff
@@ -206,3 +206,13 @@
   docsUrl: https://www.krakend.io/docs/telemetry/opentelemetry/
   components: [Go]
   oss: true
+- name: Azure SDKs
+  url: https://github.com/Azure/azure-sdk-for-net
+  docsUrl: https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-enable?tabs=aspnetcore
```
Comment (svrnm) [inline data/ecosystem/integrations.yaml:211]:
```suggestion
  docsUrl: https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-enable
```

**Review by svrnm — CHANGES_REQUESTED** [review]

**Review by theletterf — APPROVED** [review]
Approved once @svrnm suggestions are integrated. Thanks!

**Review by svrnm — APPROVED** [review]

**Pull request merged**